README.md

```diff
@@ -97,6 +97,9 @@ A list of companies that have paid Developer Community Writer Programs.
 
 - [Linode](https://www.linode.com/docs/contribute/) - Up to $300 per piece
   > Technical tutorials with code on Linux or Linode.
+  
+- [Logical Clocks](https://www.logicalclocks.com/job-positions/guest-blogger) - Pay-per-tutorial setting, GPU credits
```

Comment:
hey, is there a monetary value assigned to the compensation? if yes, could you add that? We only add publications that pay writers to the list

Reply:
Hi, I'm not sure of the exact compensation they provide.
The following information is available on the website:
### Benefits

- Compensation for each blog published;
- build your brand by publishing at a world leading ML company, including your biography and photo on each blog you write;
- get free access to lots of GPUs on the Hopsworks platform.

Reply:
@balapriyac any word on how much they compensate writers?

We'd prefer to only include sources with set payment rates. Thanks!

Reply:
Hi @karllhughes, 
I'm sorry. I couldn't find their pay per article. Kindly close the PR.